README.rst

```diff
@@ -384,6 +384,12 @@ TMT_FORCE_COLOR
     and ``TMT_NO_COLOR``. If user tries both to disable and enable
     colorization, output would be colorized.
 
+TMT_SHOW_TRACEBACK
+    By default, when tmt reports an error, the corresponding
+    traceback is not printed out. When ``TMT_SHOW_TRACEBACK`` is
+    set to any string except ``0``, tracebac would be printed out.
+    If set to anything but ``0``, when tmt reports an error, its
```

Comment:
Typo above (tracebac) and this line is duplicate, right?

Reply:
Correct, fixed in d7e5ab00.

Reply:
The `tracebac` typo not fixed.

Reply:
Oops, sorry, fixed in e42f5708.